inst/components/scss/sidebar.scss

```
@@ -38,6 +40,15 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--_icon-size)), var(--_sideba
   border: var(--_border);
   border-radius: var(--_border-radius);
 
+  // Ensure that the layout is tall enough to accommodate all toggle buttons
+  &[data-bslib-sidebar-open="open"],
+  &[data-bslib-sidebar-open="desktop"],
+  &[data-bslib-sidebar-open="closed"] {
+    &, &.html-fill-item {
+      min-height: var(--_toggle-collective-height);
+    }
+  }
```

I like this is precisely correct, but part of me wants to do this for simplicity sake? Also, do you need the `&.html-fill-item` to override htmltools' css?

```suggestion
  // Ensure that the layout is tall enough to accommodate all toggle buttons
  &, &.html-fill-item {
      min-height: var(--_toggle-collective-height);
    }
```

---

Yeah the counter might also not exist in the always case, and then the default would be sensible. I'll investigate.

---

The `fill.css` rules include a rule like this:

```css
.html-fill-container > .html-fill-item {
  flex: 1 1 auto;
  min-height: 0;
  min-width: 0;
}
```

This example points out that we might want to do something like this instead:

```css
.html-fill-container > .html-fill-item {
  flex: 1 1 auto;
}

:where(.html-fill-container > .html-fill-item) {
  min-height: 0;
  min-width: 0;
}
```

The problem is that, as currently written, the specificity of the `min-height` rule is relatively high, so setting a `min-height` on the `.bslib-sidebar-layout` class isn't enough, we basically have to set it on something with a strictly higher specificity, like `.html-fill-container > .bslib-sidebar-layout.html-fill-item`. That's... not great.

If we instead wrap the whole rule in `:where()` we demote the specificity low enough that a single class rule can set `min-height`, which seems reasonable given the objective of the fill rules.

I'm going to make this work for now but leave a comment (and we can address in the fill rules if we want).